integration-tests/spring-data-jpa/src/main/resources/application.properties

```diff
@@ -8,3 +8,5 @@ quarkus.hibernate-orm.database.generation=drop-and-create
 #quarkus.hibernate-orm.log.sql=true
 
 %prod.quarkus.hibernate-orm.sql-load-script=import.sql
+
+quarkus.log.category.\"io.quarkus.spring.data\".level=TRACE
```

Comment:
I doubt we wan't this :)

Reply:
woops!

Reply:
It seems to still be there, can you remove it please?

Reply:
Crap. I thought I had. I bet it got lost when I got frustrated with messy attempts and just reset things back to what I'd pushed already in a fit of pique.